packages/react-component-library/src/components/AutocompleteE/AutocompleteEOption.tsx

```diff
@@ -3,11 +3,28 @@ import React from 'react'
 import { SelectBaseOption, SelectBaseOptionProps } from '../SelectBase'
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface AutocompleteEOptionProps extends SelectBaseOptionProps {}
+export interface AutocompleteEOptionProps extends SelectBaseOptionProps {
+  inputValue?: string
+}
+
+function emboldenInputValue(children: string, inputValue: string) {
+  if (inputValue) {
+    const regex = new RegExp(`(${inputValue})`, 'gi')
+    return children.replace(regex, '<strong>$1</strong>')
+  }
+
+  return children
```

Comment:
Need to HTML escape this one as well:

```suggestion
  return escapeHTML(children)
```

(Can test by making an option e.g. `&lt;b&gt;thirty`).